src/tx/tx.constants.ts

```diff
@@ -3,15 +3,17 @@ import type { Currency } from '@planetarium/tx';
 import { Address } from '@planetarium/account';
 
 // FIXME: Get NCG_MINTER from external source (e.g., environment varibles).
-export const NCG_MINTER = Address.fromHex(process.env.NCG_MINTER);
+export const NCG_MINTER = process.env.NCG_MINTER
+  ? Address.fromHex(process.env.NCG_MINTER)
+  : null;
 
 export const SUPER_FUTURE_DATETIME = new Date(2200, 12, 31, 23, 59, 59, 999);
 
 export const CURRENCIES: Record<string, Currency> = {
   NCG: {
     ticker: 'NCG',
     decimalPlaces: 2,
-    minters: new Set([NCG_MINTER.toBytes()]),
+    minters: NCG_MINTER ? new Set([NCG_MINTER.toBytes()]) : null,
```

Comment:
Is it should be empty list or null? @moreal

Reply:
It should be `null` instead of `empty list`. Ther are different.